rir/src/compiler/translations/pir_translator.cpp

```diff
@@ -1,9 +1,11 @@
 #include "pir_translator.h"
+#include "rir_compiler.h"
 
 namespace rir {
 namespace pir {
 
-bool PirTranslator::isVerbose() { return verbose; }
-void PirTranslator::setVerbose(bool isVerbose) { verbose = isVerbose; }
+void PirTranslator::apply(Closure* function) { 
+    this->applyTranslation(function);
```

Comment:
I don't understand the purpose of this delegation. Why not call all of them apply, or applyTranslation

(no `this` required)

Reply:
Good catch. Completely nonsense what I did.